md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTemplateAPI.java

```diff
@@ -454,31 +397,25 @@ RestWrapper apply(@ModelAttribute("processtemplate")
                     LOGGER.debug("processtempinfo.canrecover= " + processTempInfo.getCanRecover());
                     LOGGER.debug("processInfo next process id is " + processInfo.getNextProcessIds());
                     processTempInfo.setNextProcessIds(processInfo.getNextProcessIds());
-                    //s.selectOne("call_procedures.UpdateProcess", processTempInfo);
                 }
             }
             // Updating existing properties with changes made in properties template
             for (ProcessTemplate processTempInfo : processTemplateInfos) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "processTempInfo" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)